e2etests/test/method_disconnect.js

```diff
@@ -171,30 +171,27 @@ protocolAndTermination.forEach( function (testConfiguration) {
     };
 
     var sendMethodCall = function(serviceClient, methodName, testPayload, sendMethodCallback) {
-      setTimeout(function() {
-        // make the method call via the service
         var methodParams = {
           methodName: methodName,
           payload: testPayload,
           connectTimeoutInSeconds: 10,
           responseTimeoutInSeconds: 10
         };
-        debug('service sending method call:');
+        debug('service invoking method: ' + methodName + ' with payload:');
         debug(JSON.stringify(methodParams, null, 2));
         serviceClient.invokeDeviceMethod(
-                provisionedDevice.deviceId,
-                methodParams,
-                function(err, result) {
-                  if(!err) {
-                    debug('got method results');
-                    debug(JSON.stringify(result, null, 2));
-                    assert.strictEqual(result.status, methodResult);
-                    assert.deepEqual(result.payload, testPayload);
-                  }
-                  debug('---------- end method test ------------');
-                  sendMethodCallback(err);
-                });
-      }, 1000);
+          provisionedDevice.deviceId,
+          methodParams,
+          function(err, result) {
+            if(!err) {
+              debug('got method results');
+              debug(JSON.stringify(result, null, 2));
+              assert.strictEqual(result.status, methodResult);
```

Comment:
>result.status, methodResult [](start = 33, length = 27)

I would like to add a 'policy' to e2e tests as much as possible.  I want to see the device id printed out with every failure.  This way we can examine the logs and have the device id as one of the search parameters.

Reply:
that's a good idea. let's start doing that.